content/en/docs/contributing/code/_index.md

```diff
@@ -584,6 +584,31 @@ If you want to debug using `jx` with `stdin` to test out terminal interaction, y
 	dlv --listen=:2345 --headless=true --api-version=2 attach SomePID
 	```
 
+### Debugging jx boot pipeline
+
+If you want to debug a pipeline, for example `install-vault` pipeline you'll have to change the args and command in `jenkins-x.yml` file in the boot config directory:
```

Comment:
You mean debugging local execution, not in cluster, right? Might be worth clarifying.